lib/license.js

```diff
@@ -144,7 +207,11 @@ function writeTemplatedLicense(newName, oldName, text, pkgName) {
   assert(oldName);
   assert(text);
   assert(pkgName);
-  var license = _.template(String(text))({name: pkgName});
+  var others = hasOthers() ? ' and other contributors' : '';
```

Comment:
`Copyright (c) 2013-2015 StrongLoop, Inc<%= others %>.` + `' and other contributors'`
`Copyright (c) 2013-2015 StrongLoop, Inc and other contributors.`

Should be `Copyright (c) 2013-2015 StrongLoop, Inc. and other contributors.`